docs/agent/cli.mdx

```diff
@@ -191,7 +191,7 @@ ngrok config add-api-key 1roPsn7AascHeO18mHcxRD3xT76_3ww7C9CDLYNgcdSYsscCB
 
 The `ngrok config add-authtoken` command saves the ngrok authtoken to the configuration file. You can find your authtoken in the [getting started section of the ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken).
 
-The ngrok service requires that you [sign up for an account](https://dashboard.ngrok.com/) to use many advanced service features. In order to associate your agent with an account, it must pass a secret token to the ngrok service when it starts up. Instead of passing this authtoken on every invocation, you may use this command to save it into your configuration file so that your agent always authenticates you properly.
+The ngrok service requires that you [sign up for an account](https://dashboard.ngrok.com/) to connect with an agent and some advanced service features require a paid account. In order to associate your agent with an account, it must pass a secret token to the ngrok service when it starts up. Instead of passing this authtoken on every invocation, you may use this command to save it into your configuration file so that your agent always authenticates you properly.
```

Comment:
Maybe should put a break between the connect bit and the advance+paid bit? Like: `...to connect with an agent. Some advanced service features require a paid account.`

Reply:
Pushed an updated commit